setup.cfg

```diff
@@ -16,7 +16,7 @@ extend-ignore =
     F841  # Unused variables are checked by pylint
 
 [tool:pytest]
-addopts = -n auto -rfE -s --strict-markers
+adopts = -n auto -rfE -s --strict-markers
```

Comment:
pretty sure this should be `addopts`: http://doc.pytest.org/en/latest/customize.html#adding-default-options

Reply:
ok